src/NewsService.Models.Dto/Models/NewsInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace LT.DigitalOffice.NewsService.Models.Dto.Models
+{
+  public record NewsInfo
```

Comment:
NewsPreviewInfo?

Reply:
and maybe here should be News id?

Reply:
NewsInfo is correct, but service is old, it will be necessary to discuss and redo some points